test/helpers/common.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiPromise } from "@polkadot/api";
+import { u32 } from "@polkadot/types";
+import Bottleneck from "bottleneck";
+
+export function rateLimiter() {
```

Comment:
I think those should be a feature of moonwall no ?

Reply:
This is project specific so wanted to keep this alongside the tests. A different network/team might have access to different infra and so different request parameters will suffice